eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -204,12 +211,11 @@ decodeAddress28 stakeRef (Addr28Extra a b c d) = do
       addrHash =
         hashFromPackedBytes $
           PackedBytes28 a b c (fromIntegral (d `shiftR` 32))
-  pure $ Addr network paymentCred (StakeRefBase stakeRef)
+  pure $! Addr network paymentCred (StakeRefBase stakeRef)
```

Comment:
We think this bang could be anything to do with the 1.31 regression?

Reply:
I doubt, but I was thinking this too. So, who knows, maybe.